kotlin/anoncreds/src/commonTest/kotlin/TestAnoncreds.kt

```diff
@@ -9,13 +11,33 @@ import anoncreds_uniffi.Schema
 import anoncreds_uniffi.Verifier
 import anoncreds_uniffi.createLinkSecret
 import kotlinx.serialization.json.*
+import kotlin.test.AfterTest
+import kotlin.test.BeforeTest
 import kotlin.test.Test
 import kotlin.test.assertTrue
 
 class TestAnoncreds {
-    val verifier = Verifier()
-    val issuer = Issuer()
-    val prover = Prover()
+    private var verifier: Verifier? = null
+    private var issuer: Issuer? = null
+    private var prover: Prover? = null
+
+    private var ffiObjects: MutableList<Disposable>? = null
```

Comment:
This can be initialized as an empty list by `MutableListOf<Disposable>()` to prevent it from being optional.